release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3136,4 +3136,23 @@ $ oc adm release info 4.12.1 --pullspecs
 [id="ocp-4-12-1-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-2"]
+=== RHSA-2023:0569 - {product-title} 4.12.2 bug fix and security update
+
+Issued: 2022-02-07
+
+{product-title} release 4.12.2, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0569[RHSA-2023:0569] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:0568[RHBA-2023:0568] advisory.
```

Comment:
Just double checking. Will RHSA-2023 :0569 go live soon. Currently, I get a 404.

Reply:
It will go live once everything ships.